bin/job

```diff
@@ -9,7 +9,8 @@ echo "$@"
 DSTD=${1:-/v2v-dst}
 SRCTYPE=${2}
 SRC=${3}
-LIBVIRTSRCURI=${4}
+SRCURI=${4}
+SRCPASS=${5}
```

Comment:
the password will be a parameter of the job I guess?

Unsafe, but works ...

Reply:
We could pass a file somehow. `virt-v2v` accepts only a file so we need to pass it.

Reply:
I think the parameter is the most convenient right now.

Usually Secret objects are used to pass in secrets to pods